docs/sources/talks-and-articles/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Further reading"
+menuTitle: "Further reading"
+description: "Talks and articles about Grafana Mimir"
+weight: 40
+keywords:
+  - Grafana Mimir talks and articles
+---
+
+# Furhter reading
+
+To learn more about Grafana Mimir, consul the following talks and articles.
+
+- March 2022 blog post "[Announcing Grafana Mimir, the most scalable open source TSDB in the world](https://grafana.com/blog/2022/03/30/announcing-grafana-mimir/)"
```

Comment:
What do you think about an anti-chronological order? Most recent, first.

Reply:
Good idea.